src/epub/text/book-1.xhtml

```diff
@@ -356,7 +356,7 @@
 				<br/>
 				<span>And gave her feet a footstool. Near to her</span>
 				<br/>
-				<span>He drew his parti-colored seat, aloof</span>
```

Comment:
This was probably changed by our team to `parti-colored` because that's the [usual spelling](https://www.merriam-webster.com/dictionary/parti-color). Can you remove this from the PR? Thanks!

Reply:
Ah, sorry! I've reverted to "parti-colored" and pushed another commit. Let me know if you'd rather I squash the two commits into one (by rebasing a force-pushing).